packages/stores/src/ui-config/manage-liquidity/add-concentrated-liquidity.ts

```diff
@@ -463,138 +465,148 @@ export class ObservableAddConcentratedLiquidityConfig {
     let initialized = false;
 
     // Set the initial range to be the initial custom range
-    autorun(() => {
-      if (
-        this.pool &&
-        !initialized &&
-        this._baseDepositAmountIn.sendCurrency &&
-        this._quoteDepositAmountIn.sendCurrency
-      ) {
-        initialized = true;
-
-        const multiplicationQuoteOverBase = DecUtils.getTenExponentN(
-          (this._baseDepositAmountIn.sendCurrency.coinDecimals ?? 0) -
-            (this._quoteDepositAmountIn.sendCurrency.coinDecimals ?? 0)
-        );
-
-        // Set the initial range to be the moderate range
-        this.setMinRange(
-          this.initialCustomPriceRange[0]
-            .mul(multiplicationQuoteOverBase)
-            .toString()
-        );
-        this.setMaxRange(
-          this.initialCustomPriceRange[1]
-            .mul(multiplicationQuoteOverBase)
-            .toString()
-        );
-      }
-    });
+    this._disposers.push(
+      autorun(() => {
```

Comment:
I'm not saying we should do this in this PR, but can you think of any way where we can merge this process into one call? Something like `this.extendedClass.addDisposable(() => { ... })`. I would love to avoid instances where devs forget to add to a disposer unintentionally.

Reply:
All the autoruns? Some autoruns don't need to be disposed since they live on the root store context.  Those that live in a component subtree need to be disposed if navigated away.

And if you meant that the autoruns/reactions need to be combined, that could introduce issues as we only want to react to certail things.